### ISSUE TITLE
rss feed of updated resumes

### ISSUE THREAD

**Issue by chovy** [jsonresume/registry-server#35]
I would like to get an RSS feed of recently updated resumes, given a keyword search:

For example, an RSS feed of all recently updated resumes with the word "javascript" in them.

Or if there's another flag like "currently looking" + keyword. My goal is to get a feed of resumes for people with a specific skillset that are currently looking for a job.

**Comment by thomasdavis** [comment]
Funnily enough, I was thinking of building this just a few hours ago.

I wonder if I should just make it an API call though as opposed to RSS feed, porting between either should be relatively easy.

The keyword search would be extra work but I could make a simple stream of new users and their updates relativly quickly.

**Comment by anthonyettinger** [comment]
It would be cool to have a `.rss` option, as I'm considering using this in an RSS reader, but you could easily do a JSON api as well.

**Comment by thomasdavis** [comment]
```
[{
  timestamp: 1239129391,
  username: 'thomasdavis',
  event: 'updated' // or created
}]
```

I can build that out right now and add stuff as we go.
```

**Comment by anthonyettinger** [comment]
Yeah, i would definitely want to filter the feed by keyword. Otherwise its sort of useless. Something like:

`/api/recently-updated?keywords=php+javascript&location=lat/long&radius=30miles` would be ideal as far as usefulness.

**Comment by thomasdavis** [comment]
Know any cool way to do full text search on an object and nested properties? =D

**Comment by thomasdavis** [comment]
The alternative is to pump all the resumes into algolia.com and expose the Algolia API...

**Comment by anthonyettinger** [comment]
I think this is exactly what ElasticSearch is for, searching objects

**Comment by thomasdavis** [comment]
So instead of an API endpoint for recently-updated, I will just implement a resumes end point that can be ordered by `updated_at` and filtered by `keywords` which will return the full resumes for now.

We don't have any documentation on the ethics of allowing peoples resumes to be searched at the moment but we've always had private resumes so users can switch to private if they like.

**Comment by anthonyettinger** [comment]
yeah, i would think public vs. private flag would be enough. Anyway, if the resume is viewable online it can be scraped, so I don't think there is any expectation of privacy unless they explicitly tell it to be private.
